dev-ruby/stream/stream-0.5.5.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby30 ruby31"
+RUBY_FAKEGEM_EXTRADOC="README.rdoc"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Defines an interface for external iterators for Ruby"
+HOMEPAGE="https://github.com/monora/stream"
+SRC_URI="https://github.com/monora/stream/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Ruby"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+ruby_add_bdepend "
+	test? (
+		dev-ruby/rake
+		dev-ruby/rdoc
+		dev-ruby/test-unit
+		dev-ruby/yard
```

Comment:
yard is not needed for the tests themselves but mentioned in the Rakefile. I'd either make this an unconditional bdepend (so that USE=doc also works), or alternatively to butcher the Rakefile to avoid yard altogether (more work, not needed for inclusion in gentoo).